composer.json

```diff
@@ -90,6 +89,9 @@
     "extra": {
         "branch-alias": {
             "dev-master": "3.0.x-dev"
+        },
+        "config-plugin": {
```

Comment:
Why do we need it?

Reply:
So that it can be injected by DI when using Yii

Reply:
OK.